Documentation/Wiki/Advanced-Features/Mounts.md

```diff
@@ -41,34 +34,30 @@ config({
 });
 ```
 
-### Default mounts
+### Mount permissions
+The isReadable and isWritable mount settings do not automatically give all pips access to read/write files without specifying those accesses. They are more like rule for what pips are allowed to specify. For example the `pictures` mount above has the `isReadable` property set to true. That means you may define a pip that reads from a path under this mount. However if a pip does not specify the read it makes under pictures, it will still get a Disallowed File Access at runtime.

```

Comment:
```suggestion
The `isReadable` and `isWritable` mount settings do not automatically give all pips access to read/write files without specifying those accesses. They are more like rule for what pips are allowed to specify. For example the `pictures` mount above has the `isReadable` property set to true. That means you may define a pip that reads from a path under this mount. However if a pip does not specify the read it makes under pictures, it will still get a Disallowed File Access at runtime.
```